easybuild/easyconfigs/t/TensorRT/TensorRT-8.6.1-foss-2022a-CUDA-11.7.0-Python-3.10.4.eb

```diff
@@ -0,0 +1,27 @@
+
+name = 'TensorRT'
+version = '8.6.1'
+versionsuffix = '-CUDA-%(cudaver)s-Python-%(pyver)s'
```

Comment:
Please remove the `-Python-%(pyver)s`  (we don't use it anymore for the default python in each toolchain generation)